macos/mac_os_resource_snd.ksy

```diff
@@ -0,0 +1,348 @@
+        type: u4
+        doc: pointer to audio info
+  compressed:
+    seq:
+      - id: format
+        size: 4
+        type: str
+        encoding: ASCII
+        doc: data format type
+      - id: reserved
+        -orig-id: futureUse2
+        size: 4
+      - id: state_vars_ptr
+        -orig-id: stateVars
+        type: u4
+        doc: pointer to StateBlock
+      - id: left_over_samples_ptr
+        -orig-id: leftOverSamples
+        type: u4
+        doc: pointer to LeftOverBlock
+      - id: compression_id
+        -orig-id: compressionID
+        type: s2
+        doc: ID of compression algorithm
+      - id: packet_size
+        -orig-id: packetSize
+        type: u2
+        doc: number of bits per packet
+      - id: synthesizer_id
+        -orig-id: snthID
+        type: u2
+        doc: |
+          Latest Sound Manager documentation specifies this field as:
+          This field is unused. You should set it to 0.
+          Inside Macintosh (Volume VI, 1991) specifies it as:
+          Indicates the resource ID number of the 'snth' resource that was used to compress the packets contained in the compressed sound header.
+        doc-ref: "https://vintageapple.org/inside_o/pdf/Inside_Macintosh_Volume_VI_1991.pdf Page 22-49, absolute page number 1169 in the PDF"
+    instances:
+      compression_type:
+        value: compression_id
+        enum: compression_type_enum
+  unsigned_fixed_point:
+    seq:
+      - id: integer_part
+        type: u2
+      - id: fraction_part
+        type: u2
+    instances:
+      value:
+        value: integer_part + fraction_part/65535.0
+enums:
+  data_type:
+    0x01: square_wave_synth
+    0x03: wave_table_synth
+    0x05: sampled_synth
+  wave_init_option:
+    0x04:
+      id: channel0
+      -orig-id: waveInitChannel0
+      doc: Play sounds through the first wave-table channel
+    0x05:
+      id: channel1
+      -orig-id: waveInitChannel1
+      doc: Play sounds through the second wave-table channel
+    0x06:
+      id: channel2
+      -orig-id: waveInitChannel2
+      doc: Play sounds through the third wave-table channel
+    0x07:
+      id: channel3
+      -orig-id: waveInitChannel3
+      doc: Play sounds through the fourth wave-table channel
+  init_option:
+    0x0002:
+      id: chan_left
+      -orig-id: initChanLeft
+      doc: left stereo channel
+    0x0003:
+      id: chan_right
+      -orig-id: initChanRight
+      doc: right stereo channel
+    0x0004:
+      id: no_interp
+      -orig-id: initNoInterp
+      doc: no linear interpolation
+    0x0008:
+      id: no_drop
+      -orig-id: initNoDrop
+      doc: no drop-sample conversion
+    0x0080:
+      id: mono
+      -orig-id: initMono
+      doc: monophonic channel
+    0x00C0:
+      id: stereo
+      -orig-id: initStereo
+      doc: stereo channel
+    0x0300:
+      id: mace3
+      -orig-id: initMACE3
+      doc: MACE 3:1
+    0x0400:
+      id: mace6
+      -orig-id: initMACE6
+      doc: MACE 6:1
+
+  cmd_type:
```

Comment:
Maybe these enum values deserve adding `doc` descriptions from page 2-12 [in the PDF](https://developer.apple.com/library/archive/documentation/mac/pdf/Sound/Sound_Manager.pdf)?